### ISSUE TITLE
MongoDB > v3.0:  skip' field must be numeric.

### ISSUE THREAD

**Issue by cupofnestor** [forms-angular/forms-angular#77]
I'm attempting to dockerize my fng projects.  When using fng with the latest mongodb, attempting to 'list' my models returns the error:

>Invalid query

>Failed to parse: { find: "leaders", filter: {}, projection: {}, skip: {}, limit: 60 }. 'skip' field must be numeric.

I tracked this down to an [assertion](https://github.com/forms-angular/forms-angular/blob/master/dist/forms-angular.js#L3560) in forms-angular.js:
```
if (value && value !== ''  ) {
     ///build querystring
}
```

If the skip parameter happens to be `0` (Boolean(0) == false), this assertion will fail, and no skip parameter will be added to the querystring.  I modified the assertion and the error is gone:

```
if (value && value !== '' || typeof(value) == "number" ) {
     ///build querystring
}
```

**Comment by mchapman** [comment]
Good spot, thank you.  Would you be able to submit a PR?  (Not because I am lazy, but because there is a perceived value in having contributors).

**Comment by cupofnestor** [comment]
Sure, will do.  I might submit a PR with my Dockerfile eventually, maybe that is a convo for gitter though.

**Comment by mchapman** [comment]
Closed by #78

**Comment by omererbil** [comment]
if u want pass a variable on skip or limit just do this
var i=Number(req.query.name)
.skip(i)
